ocaml/xapi/message_forwarding.ml

```diff
@@ -988,14 +988,17 @@ functor
         Xapi_pool_helpers.with_pool_operation ~__context
           ~doc:"Pool.enable_tls_verification" ~self ~op:`tls_verification_enable
           (fun () ->
+            debug "Pool.enable_tls_verification start ... (1/3)" ;
             Cert_distrib.exchange_certificates_among_all_members ~__context ;
             all_hosts
             |> List.iter (fun host ->
                    do_op_on ~local_fn ~__context ~host (fun session_id rpc ->
                        Client.Pool.enable_tls_verification rpc session_id
-                   )
+                   ) ;
+                   debug "Pool.enable_tls_verification continuing ...  (2/3)"
```

Comment:
This will print (2/3) n times. Maybe you want to say instead `"enabling on host=%s" host`